src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -64,6 +64,9 @@ def load_arguments(self, _):
     deployment_what_if_confirmation_type = CLIArgumentType(options_list=['--confirm-with-what-if', '-c'], action='store_true',
                                                            help='Instruct the command to run deployment What-If before executing the deployment. It then prompts you to acknowledge resource changes before it continues.',
                                                            is_preview=True, min_api='2019-07-01')
+    deployment_what_if_exclude_change_types_type = CLIArgumentType(nargs="+", options_list=['--exclude-change-types', '-x'],
```

Comment:
Can we use get_enum_type() for possible values?

Reply:
Thank you @Juliehzl . I changed it to use `get_enum_type()`. Your comments are always helpful. I was not aware of that `get_enum_types()` also works with nargs (I should have tried it).